src/field_theory/adjoin.lean

```diff
@@ -386,6 +414,40 @@ begin
   { exact aeval_gen_minimal_polynomial F h }
 end
 
+/-- Algebra homomorphism `F⟮α⟯ →ₐ[F] K` are in bijection with the set of roots
+of `minimal_polynomial α` in `K`. -/
+noncomputable def alg_hom_adjoin_integral_equiv (h : is_integral F α) :
+  (F⟮α⟯ →ₐ[F] K) ≃ {x // x ∈ ((minimal_polynomial h).map (algebra_map F K)).roots} :=
```

Comment:
In fact, this holds for any algebra with a `power_basis`, of which I proved one half in PR #4986 :)

Reply:
Good point. I guess eventually the generalized result should end up in power_basis.lean